*** test_parser1.result	2020-01-25 15:54:35.677678322 +0900
--- test_parser1.output	2020-01-25 16:05:07.346678322 +0900
***************
*** 2,8 ****
  SYM i GLOBAL DEFAULT:int
  SYM j GLOBAL DEFAULT:int
  SYM c GLOBAL DEFAULT:uchar
! SYM a GLOBAL DEFAULT:ARRAY [10] of int
  SYM p GLOBAL DEFAULT:POINTER to ulong
  SYM foo FUNC DEFAULT:FUNC void (void)
  {
--- 2,8 ----
  SYM i GLOBAL DEFAULT:int
  SYM j GLOBAL DEFAULT:int
  SYM c GLOBAL DEFAULT:uchar
! SYM a GLOBAL DEFAULT:ARRAY of int
  SYM p GLOBAL DEFAULT:POINTER to ulong
  SYM foo FUNC DEFAULT:FUNC void (void)
  {
***************
*** 13,28 ****
  SYM c0 GLOBAL DEFAULT:POINTER to const int
  SYM c1 GLOBAL DEFAULT:const POINTER to int
  SYM c2 GLOBAL DEFAULT:const POINTER to const int
! SYM a0 GLOBAL DEFAULT:ARRAY [10] of POINTER to int
! SYM a1 GLOBAL DEFAULT:POINTER to ARRAY [10] of int
  SYM sca GLOBAL STATIC:ushort
  SYM scb GLOBAL EXTERN:double
  SYM cvi GLOBAL DEFAULT volatile:const int
! SYM vci GLOBAL DEFAULT volatile:const int
  SYM cci GLOBAL DEFAULT:const int
  SYM vvi GLOBAL DEFAULT volatile:int
  SYM vcvi GLOBAL DEFAULT volatile:const int
! SYM bar FUNC DEFAULT:FUNC void (volatile int x, const int y, volatile const int z)
  {
  }
- 
--- 13,27 ----
  SYM c0 GLOBAL DEFAULT:POINTER to const int
  SYM c1 GLOBAL DEFAULT:const POINTER to int
  SYM c2 GLOBAL DEFAULT:const POINTER to const int
! SYM a0 GLOBAL DEFAULT:ARRAY of POINTER to int
! SYM a1 GLOBAL DEFAULT:POINTER to ARRAY of int
  SYM sca GLOBAL STATIC:ushort
  SYM scb GLOBAL EXTERN:double
  SYM cvi GLOBAL DEFAULT volatile:const int
! SYM vci GLOBAL DEFAULT:const int
  SYM cci GLOBAL DEFAULT:const int
  SYM vvi GLOBAL DEFAULT volatile:int
  SYM vcvi GLOBAL DEFAULT volatile:const int
! SYM bar FUNC DEFAULT:FUNC void (int x, const int y, const int z)
  {
  }
